### PR TITLE
Add github.dev and VSCode.dev

### PR DESCRIPTION
**Pull request by BastelPichi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1394,6 +1394,7 @@ Table of Contents
    * [Eclipse Che](https://www.eclipse.org/che/) - Web based and Kubernetes-Native IDE for Developer Teams with multi-language support. Open Source and community driven. A online instance hosted by Red Hat is available at [workspaces.openshift.com](https://workspaces.openshift.com/).
    * [fakejson.com](https://fakejson.com/) — FakeJSON helps you quickly generate fake data using its API. Make an API request describing what you want and how you want it. The API returns it all in JSON. Speed up the go to market process for ideas and fake it till you make it.
    * [gitpod.io](https://www.gitpod.io) — Instant, ready-to-code dev environments for GitHub projects. Free for open source.
+   * [GitHub.dev](https://github.dev) Visual Studio Code in your Browser with git support.
```
Comment (ripienaar) [inline README.md:1397]: Please add this to the existing github entry that covers all the services they offer, we cannot list services multiple times the list will be thousands of lines long.

**Comment by BastelPichi** [comment]
> It's extremely easily reachable. Just press "." on any github page and the editor opens on that repo.
Yes - but people dont know to do that.

**Comment by ripienaar** [comment]
I am not listing it twice, else 100 other people will be wanring theirs listed twice as well.

**Comment by BastelPichi** [comment]
I mean than you would also have to list vscode.dev and Visual Studio together. But that doesnt make sence. The idea is completely different, and noones going to read trough the hole github feature list.

I dont care if you list it or not, you can close this PR if you want.

**Pull request closed without merging**